apps/mcserverstatus/mcserverstatus.star

```diff
@@ -0,0 +1,98 @@
+"""
+Applet: MCServerStatus
+Summary: See MC server player count
+Description: Track the player count on any MC server!
+Author: jakeva
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("cache.star", "cache")
+
+def main(config):
+    rate_cached = cache.get("mc_rate")
+
+    if rate_cached != None:
+        rate = int(rate_cached)
+
+    minecraft_edition = config.get("minecraft_edition") or 0
+    server_ip = config.get("server_ip") or "mc.hypixel.net"
+    server_port = config.get("server_port") or None
+
+    req = http.get("https://api.mcsrvstat.us/" + ("bedrock/" if minecraft_edition else "") + "2/" + server_ip + (":" + server_port if server_port else ""))
+
+    if req.status_code != 200:
```

Comment:
Would be great if the app could show some illustrative default graphic when no configuration has been passed. That enables users to see previews of the app before deciding to configure and install it.